tests/enable-gcp-services/idempotent-prune/.expected/config.yaml

```diff
@@ -0,0 +1 @@
+runCount: 2
```

Comment:
Default is 2. So we can remove this file.

Reply:
done